tools/inject-dependency-sha.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+cat composer.json | \
+  jq '.require["atk4/core"]=.require["atk4/core"]+"#'`git ls-remote git@github.com:atk4/core.git develop | cut -f1`'"' |\
+  jq '.require["atk4/data"]=.require["atk4/data"]+"#'`git ls-remote git@github.com:atk4/data.git develop | cut -f1`'"' \
```

Comment:
Add dsql as well

Reply:
dsql is not a direct dependency of atk4/ui, i'll try adding it though.

Reply:
actually - can we divert the 'dsql' dependency addition into 2.2 ?